gcc/rust/rust-system.h

```diff
@@ -36,7 +36,7 @@
 #include <map>
 #include <set>
 #include <vector>
-#include <stack>
+#include "rust-system.h" 
```

Comment:
Unless I'm wrong you're trying to include a file (`rust-system.h`) into itself, with header guard it will not expand recursively. I can't understand what you're trying to achieve here.